packages/shared/src/components/search/SearchResults/SearchResultsLayout.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {
+  PropsWithChildren,
+  ReactElement,
+  useContext,
+  useMemo,
+} from 'react';
+import { useRouter } from 'next/router';
+import { PageWidgets } from '../../utilities';
+import { StraightArrowIcon } from '../../icons';
+import {
+  providerToIconMap,
+  providerToLabelTextMap,
+  SearchProviderButton,
+} from '../SearchPanel';
+import { SearchProviderEnum, SearchSuggestion } from '../../../graphql/search';
+import { useSearchResultsLayout } from '../../../hooks/search/useSearchResultsLayout';
+import { ActiveFeedContext } from '../../../contexts';
+import { FeedItemType } from '../../cards/common';
+import { FeedItem, PostItem } from '../../../hooks/useFeed';
+import { LogEvent, Origin, TargetType } from '../../../lib/log';
+import { useLogContext } from '../../../contexts/LogContext';
+import { webappUrl } from '../../../lib/constants';
+import { SearchResultsTags } from './SearchResultsTags';
+import { SearchResultsSources } from './SearchResultsSources';
+
+type SearchResultsLayoutProps = PropsWithChildren;
+
+const AICta = {
+  Icon: providerToIconMap[SearchProviderEnum.Chat],
+  Label: providerToLabelTextMap[SearchProviderEnum.Chat],
+};
+
+function isItemPost(item: FeedItem): item is PostItem {
+  return 'post' in item && item.type === FeedItemType.Post;
+}
+
+export const SearchResultsLayout = (
+  props: SearchResultsLayoutProps,
+): ReactElement => {
+  const { children } = props;
+
+  const { isSearchResultsUpgrade } = useSearchResultsLayout();
+  const { items = [] } = useContext(ActiveFeedContext);
+  const {
+    query: { q: query },
+    push,
+  } = useRouter();
+  const { logEvent } = useLogContext();
+  const isLoading = !items.length;
+
+  const postItems = items.filter(isItemPost);
+
+  const tags = useMemo(
+    () => [...new Set(postItems.flatMap((item) => item.post.tags))],
+    [postItems],
+  );
+
+  const sources = useMemo(() => {
+    const ids = new Set<string>();
+    return postItems
+      .map((item) => item.post.source)
+      .filter((source) => {
+        if (ids.has(source.id)) {
+          return false;
+        }
+        ids.add(source.id);
+        return true;
+      });
+  }, [postItems]);
+
+  const onTagClick = (suggestion: SearchSuggestion) => {
+    const tag = suggestion.id || suggestion.title.toLowerCase();
+
+    logEvent({
+      event_name: LogEvent.Click,
+      target_type: TargetType.SearchRecommendation,
+      target_id: tag,
+      feed_item_title: tag,
+      extra: JSON.stringify({
+        origin: Origin.SearchPage,
+        provider: SearchProviderEnum.Tags,
+      }),
+    });
```

Comment:
These are not new? Please verify with Ido if he wants some other origin or tracking for this experiment (if you did not already).

Reply:
@capJavert the tags events are the same for the search suggestions as specified in the kickoff, the new ones are the `sources`. @idoshamun there are some directives for those?